src/pages/index.js

```diff
@@ -64,6 +150,9 @@ profileValidator.enableValidation();
 const addCardValidator = new FormValidator(config, "#add-card-form");
 addCardValidator.enableValidation();
 
+const avatarValidator = new FormValidator(config, "#avatar-edit-modal");
+avatarValidator.enableValidation();
+
 //eventListeners
 profileEditButton.addEventListener("click", (event) => {
   profileValidator.disableButtonState();
```

Comment:
**Might be better**

You could make a special method `resetValidation` for clearing errors and controlling the submit button.
It could look something like this:

```js

resetValidation() {
      this._toggleButtonState(); <== controlling the submit button ==

      this._inputList.forEach((inputElement) => {
        this._hideError(inputElement) <== clearing errors ==
      });

    }



```

And you can call it in `index.js` when you click on open buttons